app/components/agenda/agendaitem/agendaitem-decision.js

```diff
@@ -405,6 +405,18 @@ export default class AgendaitemDecisionComponent extends Component {
     beslissingPiecePart,
     betreftPiecePart
   ) {
+    if (!report.id) {
```

Comment:
not having a report id means new file?.
only set `decisionActivity.decisionresultCode` when none exists might be better if a result code was already set manually.